dist/src/main/java/io/camunda/application/commons/backup/BackupConfig.java

```diff
@@ -60,6 +61,18 @@ && validRepoName(tasklistBackup.getRepositoryName())
     }
   }
 
+  @Bean("backupThreadPoolExecutor")
+  public ThreadPoolTaskExecutor getTaskExecutor() {
+    final ThreadPoolTaskExecutor executor = new ThreadPoolTaskExecutor();
+    executor.setCorePoolSize(1);
+    executor.setMaxPoolSize(1);
+    executor.setThreadNamePrefix("webapps_backup_");
+    // TODO why just 6?
+    executor.setQueueCapacity(6);
```

Comment:
Should we leave 6? not sure why such a small number has been selected for a queue

Reply:
It's a very good question, I have no idea. Let's ask in the `-tech` channel.